src/components/DeleteModal/modal.scss

```diff
@@ -0,0 +1,77 @@
+$purple-nav: #8932b4;
+$deep-purple-grad-2: rgba(60, 16, 83, 0.9);
+$deep-purple-grad: rgba(60, 16, 83, 0.7);
+$light-purple-grad: rgba(173, 83, 137, 0.7);
+$deep-purple: hsl(279, 68%, 19%);
+$white: #fff;
+$red: #f00;
+
+.Overlay {  // sass-lint:disable-line class-name-format
```

Comment:
@chingsley I have also previously mentioned a better option to inline ESLint disabling comments